### DISCUSSION TITLE
Distributed Lock Manager funktionality?

### DISCUSSION THREAD

**Discussion by aadrian** [babashka/babashka#1561] [General]
Hi,

Is it possible to achieve in a "babashka way" a sort of  "Distributed Lock Manager" functionality for scripts/tasks/jobs?

E.g. this tool https://github.com/tiian/flom does a great job helping **synchronize the execution of commands, shell scripts and custom programs** .  Unfortunately being dynamically linked (and no pre-built) makes it very fragile and  unsuitable for most production environments.

Thank you.

**Comment by borkdude** [comment]
Perhaps a babashka pod can be built to accomodate such a use case.

See:

https://github.com/babashka/pod-registry

**Reply by aadrian** [reply]
> Perhaps a babashka pod 

Sure,  I understand that a POD is needed since the BB core can't be made bigger.

But my question is if there's some BB (or BB pod) way or functionality to achieve what FLoM is achieving: being able to sync the execution of BB tasks and scripts locally, but also across machines?
